service_crategen/src/commands/generate/codegen/rest_json.rs

```diff
@@ -144,6 +145,16 @@ fn generate_headers(service: &Service) -> Option<String> {
     None
 }
 
+// SageMaker Runtime allows to overwrite content-type
+fn generate_default_headers(service: &Service) -> Option<String> {
```

Comment:
Since there's no code path that returns `None` can we change the return type of this function to `String`? This will also let us remove the `unwrap_or_else` on line 95.

Reply:
Sure thing, let me do this. I just tried to mimic the style of the code around.